packages/angular_devkit/build_ng_packagr/package.json

```diff
@@ -14,6 +14,6 @@
     "rxjs": "^6.0.0"
   },
   "peerDependencies": {
-    "ng-packagr": "^2.2.0 || ^3.0.0-rc.0"
+    "ng-packagr": "^3.0.0"
```

Comment:
Maybe also add the peerDependency to the RC of V4?

Reply:
I was thinking about the peerDeps a bit more, and I think they should be these instead:
```
"peerDependencies": {
     "^2.2.0 || ^3.0.0 || ^4.0.0-rc.0"
}
```

And that the `fix(@angular-devkit/build-ng-packagr): update to ng-packagr v3` commit should be `feat` instead of `fix`.

The reasoning is that changing peerdep support changes how the package behaves, so for users on ng-packagr@2.2.0 they could update build-ng-packagr to the next patch version and get a peerdep warning/error.

This way support for 3.0 and 4 RC is added, support for 2.2 is retained, and users won't get the new build-ng-packagr peerdep errors on a patch.

WDYT?

Reply:
Removing support for 2.2.0 would be a breaking change.  Adding support for 4.0.0-rc.0 makes sense.  And then with CLI 7.0, support for 2.x (and potentially 3.x) can be dropped.